### PR TITLE
Adding intro exercises.

### LINKED ISSUE #6: Move download data to its own day 2 setup

### PR DESCRIPTION
**Pull request by ChristinaLK**
Maybe went a bit overboard.
Closes #6

### CODE REVIEW

**Comment by gonuke** [comment]
This has a conflict in the lunch episode - which may not matter, but should be resolved...

**Comment by ChristinaLK** [comment]
updated!

File: _episodes/21-data-download.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: "Introductory Exercises"
+teaching: 0
+exercises: 25
+questions:
+- "How can I run Python programs?"
```
Comment (gonuke) [inline _episodes/21-data-download.md]: Is this the right question?

**Review by gonuke — CHANGES_REQUESTED** [review]
Confirm the question in the metadata...

File: _episodes/21-data-download.md (hunk 1 above)
Comment (ChristinaLK) [reply _episodes/21-data-download.md]: oops, no it isn't.

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Looks good - thanks!

**Pull request merged**